src/main/pages/setup-kubernetes/kubernetes-config.adoc

```diff
@@ -1,41 +1,33 @@
 ---
-title: "Configuration: Kuberentes"
+title: "Configuring Kuberentes"
 keywords: kubernetes, configuration
 tags: [installation, kubernetes]
 sidebar: che_6_docs
 permalink: kubernetes-config.html
 folder: setup-kubernetes
 ---
 
-[id="how-it-works"]
-== How It Works
+You can configure the behavior of the Che server by passing environment variables to the Che deployment.
 
-Che server behavior can be configured by passing environment variables to Che deployment.
+There are multiple ways to edit the Che deployment to add new or edit existing environment variables:
 
-There are multiple ways to edit Che deployment to add new or edit existing envs:
-
-* `kubectl edit dc/che` opens Che deployment yaml in nano editor (VIM is used by default)
-* manually in Kubernetes web console > deployments > Che > Edit
-* `kubectl set env dc/che KEY=VALUE KEY1=VALUE1` updates Che deployment with new envs or modifies values of existing ones
+* To open the Che deployment YAML file in the *nano* editor, use the `kubectl edit dc/che` command. The *Vim* editor is used by default.
```

Comment:
Suggest making 'The *Vim* editor is used by default.'  as the first sentence. 
Both nano and Vim should not be bold, as per https://redhat-documentation.github.io/asciidoc-markup-conventions/ only GUI elements should be marked up bold.